index.d.ts

```diff
@@ -91,9 +91,11 @@ type StdinSingleOption<
 	| CommonStdioOption<IsSync, IsArray>
 	| InputStdioOption<IsSync, IsArray>;
 
-export type StdinOption<IsSync extends boolean = boolean> =
+type StdinOptionCommon<IsSync extends boolean = boolean> =
 	| StdinSingleOption<IsSync, false>
 	| Array<StdinSingleOption<IsSync, true>>;
+export type StdinOption = StdinOptionCommon<false>;
+export type StdinOptionSync = StdinOptionCommon<true>;
```

Comment:
Use the same async vs sync (instead of using a generic parameter) export convention as the other types.